### DISCUSSION TITLE
Live tailing: Improve UX so it is more clear and obvious that when you scroll, live tailing is paused

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#57349] [Observability]
**What would you like to be added**:
It happened to me and also @cyriltovena that we didn't realize that live tailing was paused because of scrolling trough the logs. It isn't very obvious and it might seem that live tailing is broken because no new logs are coming. 

I believe it should be more clear and obvious when the live tailing is paused because of logs scrolling.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
